zerver/openapi/zulip.yaml

```diff
@@ -1430,6 +1430,18 @@ paths:
                                   type: string
                                   description: |
                                     The text content of the status message.
+                                emoji_name:
+                                  type: string
+                                  description: |
+                                    The emoji name associated with status.
+                                emoji_code:
+                                  type: string
+                                  description: |
+                                    The emoji code associated with emoji_name.
+                                reaction_type:
+                                  type: string
+                                  description: |
+                                    The reaction type associated with emoji_name.
```

Comment:
These needed a more detailed description, and a `Changes` note to indicate they're new in a new feature level.

I'm taking care of that and merging this first commit.

That said, I notice as part of this that we don't have documentation for the `/users/me/status` API endpoint, which is actually pretty important; @MSurfer20 are you up for doing that one next?  Note it'll need a `Changes` entry for feature level 86, as documented in the API changelog, and probably another for when it was introduced.

Reply:
@timabbott Coincidentally i had opened https://github.com/zulip/zulip/pull/19403 yesterday only for adding that piece of documentation, although I'd have to add the changelog info there now that this PR is merged.

Reply:
Awesome!  Posted a few quick comments on that PR.